### ISSUE TITLE
Accessibility Full conformance test pass for TSDoc - 6/16/2021  (A11ytest_tracking, Feature_Task)

### ISSUE THREAD

**Issue by Padmini171991** [microsoft/tsdoc.org-website#22]
21:[TSDoc Home page]: Horizontal scroll bar appears in 'Code' part when text spacing is enabled.[Sev2][WCAG1.4.12][WCAG2.1]
20:[TSDoc Home Page]: Screen Reader is not announcing positions for links in navigation landmark in TSDoc home page.[Sev2][WCAG1.3.1]
18:['TSDoc' home page]: Focus is not visible properly on 'Intro' link in 'TSDoc' home page.[Sev2][WCAG2.4.7]

9:[TSDoc Home page]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.[Sev2][WCAG1.4.3][Auto]
19:[Intro]:Keyboard focus is not visible on first interactive element on activating the 'Intro' link.[Sev2][WCAG2.4.3]

11:[Intro]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Code' part in 'Intro' page.[Sev2][WCAG1.4.3][Auto]
12:[Tags]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Improve this page' link in 'Tags' page.[Sev2][WCAG1.4.3][Auto]

12:[Tags]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Improve this page' link in 'Tags' page.[Sev2][WCAG1.4.3][Auto]
12:[Tags]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Improve this page' link in 'Tags' page.[Sev2][WCAG1.4.3][Auto]
16:[Playground]: H1 level heading is not defined in 'Playground' page.[Sev2][WCAG2.4.6][Auto]
15:[Playground - HTML]:Elements that have scrollable content must be accessible by keyboard.[Sev3][WCAG2.1.1][Auto]
14:[Playground]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for '@beta' text in 'Playground' page.[Sev2][WCAG1.4.3][Auto]
13:[Playground]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Statistics' text in 'Playground' page.[Sev2][WCAG1.4.3][Auto]
10:[Playground- Lines]: Ensures every form element has a label.[Sev2][WCAG1.3.1][Auto]
8:[Playground]: Text spacing is not getting adopted to menu items in ‘Choose a sample’ and 'Light Theme' combo box.[Sev2][WCAG1.4.12][WCAG2.1]
12:[Tags]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Improve this page' link in 'Tags' page.[Sev2][WCAG1.4.3][Auto]
11:[Intro]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Code' part in 'Intro' page.[Sev2][WCAG1.4.3][Auto]
11:[Intro]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Code' part in 'Intro' page.[Sev2][WCAG1.4.3][Auto]
